packtools/sps/validation/peer_review.py

```diff
@@ -0,0 +1,324 @@
+from packtools.sps.models.article_authors import Authors
+from packtools.sps.models.dates import ArticleDates
+from packtools.sps.models.peer_review import PeerReview
+from packtools.sps.models.related_articles import RelatedItems
+from packtools.sps.validation.exceptions import ValidationPeerReviewException
+from packtools.sps.validation.utils import format_response
+
+
+def get_node_id(node):
+    return f" (sub-article: {node.get('id')})" if node.get('id') else " (article: main)"
+
+
+class RelatedArticleTypePeerValidation:
+    def __init__(self, related_article_type, related_article_type_list=None):
+        self.related_article_type = related_article_type
+        self.related_article_type_list = related_article_type_list
+
+    @property
+    def related_article_type_validation(self):
+        # Para parecer como <article> além dos elementos mencionados anteriormente, adiciona-se a tag
+        # de <related-article> referenciando o artigo que sofreu o parecer. Neste caso utiliza-se:
+        # @related-article-type com valor "peer-reviewed-material";
+        if not self.related_article_type_list:
+            raise ValidationPeerReviewException("Function requires list of related articles")
+        expected = ' | '.join(self.related_article_type_list)
```

Comment:
@Rossi-Luciano expected pode ser a própria lista self.related_article_type_list, como validation_type é value in list, entende-se que o esperado é um item da lista.

Reply:
OK @robertatakenaka